src/cnd/axios_rfc7807_middleware.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { Problem } from "./axios_rfc7807_middleware";
+
+describe("problem message", () => {
+  it("should format all fields to a descriptive message", () => {
+    const problem = new Problem({
+      title: "Swap not supported.",
+      status: 400,
+      detail: "This combination of protocols is not supported.",
+      type: "https://comit.network/docs/errors/swap-not-supported"
+    });
+
+    expect(problem.message).toMatchInlineSnapshot(
```

Comment:
I am not sure to understand why you are using the snapshot feature here as `message` is just a string with no html elements.

Reply:
Because I was too lazy to write the expected message :)

I wrote the code first and I am using snapshot tests to "freeze" the implementation after I confirmed visually that this is what I want!